pkgs/applications/audio/nova-filters/default.nix

```diff
@@ -16,6 +16,7 @@ stdenv.mkDerivation rec {
     sed -i -e '4d' SConstruct
     sed -i "s@mfpmath=sse@mfpmath=sse -I ${boost.dev}/include@g" SConstruct
     sed -i "s@ladspa.h@${ladspaH}/include/ladspa.h@g" filters.cpp
+    sed -i "s@LADSPA_HINT_SAMPLE_RATE, 0, 0.5@LADSPA_HINT_SAMPLE_RATE, 0.0001, 0.5@g" filters.cpp
```

Comment:
@Mic92 I've communicated with the maintainer, and he stated he's not maintaining this anymore.
Since I'm still heavily using it, I'm patching it here.